src/Validations/admin.js

```diff
@@ -0,0 +1,57 @@
+import Joi from 'joi';
+
+const adminSchema = Joi.object({
+  firstName: Joi.string()
+    .pattern(/^[a-zA-Z]+$/)
+    .min(4)
+    .max(10)
+    .required()
+    .messages({
+      'string.base': 'First Name should be a string',
+      'string.pattern.base': 'First Name should only contain alphabetic characters',
+      'string.min': 'First Name should have a minimum length of 4',
+      'string.max': 'First Name should have a maximum length of 10',
+      'any.required': 'First Name is required'
+    }),
+  lastName: Joi.string()
+    .pattern(/^[a-zA-Z]+$/)
+    .min(4)
+    .max(10)
+    .required()
+    .messages({
+      'string.base': 'Last Name should be a string',
+      'string.pattern.base': 'Last Name should only contain alphabetic characters',
+      'string.min': 'Last Name should have a minimum length of 4',
+      'string.max': 'Last Name should have a maximum length of 10',
+      'any.required': 'Last Name is required'
+    }),
+  phone: Joi.string().min(10).required().messages({
+    'string.min': 'Phone should have a minimum length of 10',
+    'any.required': 'Phone is required'
+  }),
```

Comment:
I would write these rules to only accept numbers:
phone: Joi.string()
    .pattern(/^[0-9]+$/)
    .min(10)
    .max(15)
    .required()
    .messages({
      'string.pattern.base': 'Phone Number should only contain numbers',
      'string.base': 'Phone Number should be a string',
      'string.empty': 'Phone Number is required',
      'string.min': 'Phone Number should have a minimum length of 10',
      'string.max': 'Phone Number should have a maximum length of 15'
    }),